components/shopify_developer_app/sources/new-refund-created/new-refund-created.mjs

```diff
@@ -0,0 +1,26 @@
+import constants from "../common/constants.mjs";
+import common from "../common/webhook.mjs";
+
+export default {
+  ...common,
+  key: "shopify_developer_app-new-refund-created",
+  name: "New Refund Created (Instant)",
+  description: "Emit new event when a new redung is cerated.",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    getTopic() {
+      return constants.EVENT_TOPIC.REFUNDS_CREATE;
+    },
+    generateMeta(resource) {
+      const ts = Date.parse(resource.updated_at);
```

Comment:
The `test-event` doesn't include the field `updated_at`. I thnk it's safe to use `created_at` since the resource represents the refund itself, rather than the order.
```suggestion
      const ts = Date.parse(resource.created_at);
```